src/registry_schemas/schemas/business.json

```diff
@@ -64,6 +64,25 @@
         "XP",
         "XS"
       ]
+    },
+    "complianceWarning": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "title": "Warning code",
+          "examples": ["INVALID_LEGAL_STRUCTURE_DIRECTORS"]
+        },
+        "message": {
+          "type": "string",
+          "title": "Warning message",
+          "examples": ["A minimum of 3 directors is required"]
+        },
+        "filing": {
+          "type": "string",
+          "title": "The link to the filing that resulted in the non-compliance."
+        }
+      }
     }
   },
   "properties": {
```

Comment:
`arMaxDate` and `arMinDate` are also returned in the business call.

Reply:
Is `dissolutionDate` (on line 115 below) obsolete now?

Reply:
Sample business response:

![Uploading image.png…]()

Reply:
There are a lot of properties missing that are currently being provided...

Reply:
Yes. I noticed that. I added only the ones related to entity state.

Reply:
OK